13-students-doing-data-science.Rmd

```diff
@@ -84,60 +84,63 @@ For example,
 
 There are examples from data science books (Wickham & Grolemund, 2017) and past research (e.g., Lehrer, Kim, & Schauble, 2007; see Lehrer & Schauble, 2015, for more examples) that suggests that starting with visualizing data can be benefician in terms of learners' ability to work with data. Wickham and Grolemund (2017) write that they begin their book, *Data Science Using R*, with a chapter on visualization, because doing so allows learners to create something that they can share immediately, whereas tasks such as loading data can be rife with issues - and does not immediately lead learners to have a product they can share. Lehrer, Kim, and Schauble (2007) show how providing students with an opportunity to invent statistics by displaying the data in new ways. This led to (productive) critique among fifth- and sixth-grade students and their teacher.
 
-### Consider representation and inclusivity in your teaching
+### Consider representation in the data and examples you use
 
-This includes examples, data sets.
+One way to think about data is that it is objective, free of decisions about what to value or prioritize. Another is to consider data as a process that is value-laden, from deciding what question to ask (and what data to collect) to interpreting findings with attention to how others will make sense of them (e.g., O'Neill's *Weapons of Math Destruction*, and Lehrer, Kim, and Schauble's (2007) description of data modeling). From this broader view, choosing representative data is a choice, like others, that teachers can make. 
```

Comment:
I love this! This touches on the equity discussion we had yesterday and we can think of how to thread the needle between all the times it's brought up. I think it's worth expanding this a bit and show examples of how this sort of awareness is needed throughout the entire data science process.

Reply:
thanks much - I was proud of this (and also thought, and think, it can be expanded). Examples will help. Will work on this.

Reply:
just began to address, but added: "For example, instructors can choose data that directs attention to issues-- equity-related issues in education, for example--that she or he believes would be valuable for students to analyze."